s3_management/thirdparty_deps/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8
```

Comment:
For this, could you double check to see if `3.8-alpine` works?  The difference in size is worth the try, python:3.8 size is 861MB while 3.8-alpine is only 46MB.  I don't think we need any complicated things here besides base python and pip

Reply:
Probably makes sense to check https://github.com/pytorch/builder/blob/main/s3_management/Dockerfile as well.

Reply:
On the other hand, it may not matter, because the base image is popular and probably already cached on github runners, so only the diff will be actually pulled.

Reply:
Changed base to  `python:3.8-alpine` and verified that is works.